tests/x11/remote_desktop/windows_network_setup.pm

```diff
@@ -1,39 +1,41 @@
 # SUSE's openQA tests
 #
-# Copyright 2018-2021 SUSE LLC
+# Copyright 2018-2022 SUSE LLC
```

Comment:
please keep the old date. See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md?plain=1#L49 for this